tests/test_codecs.py

```diff
@@ -1,6 +1,9 @@
 from unittest import TestCase, mock
 
+from parameterized import parameterized
+
 from ffmpeg_tools import codecs
+from ffmpeg_tools import commands
```

Comment:
Why are this new imports here, when nothing changed?

Reply:
Same as above. I must have missed them when I was moving stuff around. I'll remove them.

Reply:
Done